homeassistant/components/foscam/camera.py

```diff
@@ -60,6 +133,12 @@ def __init__(self, device_info):
             if result == 0:
                 self._rtsp_port = response.get("rtspPort") or response.get("mediaPort")
```

Comment:
Can we move this and the request out of init? It's not allowed to do sync I/O in async context.